setup.cfg

```
@@ -0,0 +1,17 @@
+[pycodestyle]
+exclude = .eggs,ENV,build,docs/conf.py,venv
+
+[yala]
+radon mi args = --min C
+pylint args = --disable=too-few-public-methods,too-many-instance-attributes,no-name-in-module --ignored-modules=napps.amlight.sdntrace
+
```

```suggestion
linters=pylint,pycodestyle,isort
```

@rmotitsuki could you explicitly set the linters? Otherwise `yala` tries to find them, and depending if you have other installed they can catch more errors than exepcted.